src/reset-password/controller/redirect.ts

```diff
@@ -1,8 +1,45 @@
 import Controller from '@curveball/controller';
```

Comment:
Lets change the name of `redirect` to something else

Reply:
Yeah, I'd like to sit down with you and talk about naming convention for few of them. because some of them doesn't represent what the method actually does